canton/community/lib/README.md

```diff
@@ -1,6 +1,6 @@
 Forked or extended Daml and third-party libraries:
 
-- [akka](https://github.com/akka/akka/) - Forked some files for adding more debug logging
+- [pekko](https://github.com/pekko/pekko/) - Forked some files for adding more debug logging
```

Comment:
@andreaslochbihler-da @remyhaemmerle-da 
This file and the pekko/LICENCE file was only changed by my migration script.
Is this sufficient for now? (as before releasing we anyway need the proper fix for the source of these canton files, which needs to be propagated anyway to daml again)?

Reply:
this is part of the drop, so I do not care.

Reply:
It will be overwrite by the next code prod.

Reply:
Thank you